<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriadsEditorDialog</class>
 <widget class="QDialog" name="TriadsEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblIcon">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCaption">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblFileName">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;File name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFileName"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="toolTip">
        <string>Save file</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="toolTip">
        <string>Add triplet</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/zoomin</normaloff>:/icons/zoomin</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="toolTip">
        <string>Remove triplet</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/zoomout</normaloff>:/icons/zoomout</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frmTriplets">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>TriadsEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>TriadsEditorDialog</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>TriadsEditorDialog</receiver>
   <slot>onRemoveTriplet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>TriadsEditorDialog</receiver>
   <slot>onAddTriplet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>TriadsEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddTriplet()</slot>
  <slot>onRemoveTriplet()</slot>
  <slot>onSave()</slot>
 </slots>
</ui>
